### PR TITLE
Add CLI command to update validator public key

### LINKED ISSUE #539: Allow validators to change BLS key
### Expected Behavior

Allow validators to change BLS keys with the same semantics as they can change their signing key.

### Current Behavior

Can't swap out BLS keys.

### PR DESCRIPTION
**Pull request by asaj**
### Description

This PR adds a smart contract function and corresponding CLI command for validators to update their public keys.

*Note that this update will not affect the existing epoch, only future epochs.*

### Tested

Unit tests.

### Other changes

None

### Related issues

- Fixes https://github.com/celo-org/celo-blockchain/issues/539

### Backwards compatibility

Technically backwards compatible

### CODE REVIEW

**Comment by asaj** [comment]
@kevjue, we'll need to update `ValidatorSetDiff` to return a diff if the BLS public key changes. Any other celo-blockchain changes you can think of?

File: packages/protocol/contracts/governance/Validators.sol [hunk 1]
```diff
@@ -490,6 +491,31 @@ contract Validators is
...
+   * @notice Updates a validator's public keys data.
+   * @param publicKeysData Comprised of three tightly-packed elements:
+   *    - publicKey - The public key that the validator is using for consensus, should match
+   *      msg.sender. 64 bytes.
+   *    - blsPublicKey - The BLS public key that the validator is using for consensus, should pass
+   *      proof of possession. 48 bytes.
+   *    - blsPoP - The BLS public key proof of possession. 96 bytes.
+   * @return True upon success.
+   */
+  function updatePublicKeysData(bytes calldata publicKeysData) external returns (bool) {
```
Comment (nambrot) [inline packages/protocol/contracts/governance/Validators.sol:479]: Could you reuse this function in `registerValidator`?

**Review by nambrot — APPROVED** [review]
LGTM, just a minor refactor suggestion

File: packages/cli/src/commands/validator/publicKey.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  static examples = [
+    'publickey --from 0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95 --publicKey 0xc52f3fab06e22a54915a8765c4f6826090cfac5e40282b43844bf1c0df83aaa632e55b67869758f2291d1aabe0ebecc7cbf4236aaa45e3e0cfbf997eda082ae19d3e1d8f49f6b0d8e9a03d80ca07b1d24cf1cc0557bdcc04f5e17a46e35d02d0d411d956dbd5d2d2464eebd7b74ae30005d223780d785d2abc5644fac7ac29fb0e302bdc80c81a5d45018b68b1045068a4b3a4861c93037685fd0d252d7405011220a66a6257562d0c26dabf64485a1d96bad27bb1c0fd6080a75b0ec9f75b50298a2a8e04b02b2688c8104fca61fb00',
+  ]
+  async run() {
+    const res = this.parse(ValidatorPublicKey)
+    this.kit.defaultAccount = res.flags.from
+    const validators = await this.kit.contracts.getValidators()
+    const accounts = await this.kit.contracts.getAccounts()
+    await displaySendTx(
```
Comment (mcortesi) [inline packages/cli/src/commands/validator/publicKey.ts:31]: add checks?

**Review by mcortesi — APPROVED** [review]

**Pull request merged**